### ISSUE TITLE
WLED crashes (on all my boards) whenever you do anything on any version after 0.12.0

### ISSUE THREAD

**Issue by The-Techy** [Aircoookie/WLED#2909]
### What happened?

After installing a number of WLED version from 0.12.0 up, I simply cannot control the leds. I've tested on a couple ESP8266 boards with the same issue. AP opens, I connect it to wifi, can access it, can change number of leds etc. Then, as soon as I do anything, e.g. pick a colour or effect, the little dot in the bottom left corner goes red, it disconnects then reconnect a couple seconds later. The leds don't change. Happens every time, consistently, with different versions, methods of uploading/compiling, different boards, even when nothing is connected except usb. Really weird.

### To Reproduce Bug

1.Install WLED (anything above 0.012.0), using any method (web installer, 3rd party web installer, compiling and upload through VSCode etc)
2. Open AP, enter wifi details
3. Find IP and connect
4. Try and change anything

### Expected Behavior

Should change leds, just like it does on 0.12.0

### Install Method

Binary from WLED.me

### What version of WLED?

All above 0.12.0

### Which microcontroller/board are you seeing the problem on?

ESP8266

### Relevant log/trace output

```shell
Note, this did not appear consistently, this was on around the 4th crash that it showed up

--------------- CUT HERE FOR EXCEPTION DECODER ---------------
 
Exception (3):
epc1=0x40244282 epc2=0x00000000 epc3=0x00000000 excvaddr=0x400b7d61 depc=0x00000000
 
>>>stack>>>
 
ctx: sys
sp: 3fffec40 end: 3fffffb0 offset: 0190
3fffedd0:  3ffecbc1 ffff7b80 0000002a 4024441c  
3fffede0:  0000ff3f 4bc6a7f0 000148e7 3fff4e8c  
3fffedf0:  3fff4f2d 0000004f 3fff4c68 00000000  
3fffee00:  ffffffff 3fffc6fc 40101086 3fff4e8c  
3fffee10:  00000000 3fffee60 3fff6dcc 40244449  
3fffee20:  00000000 3fffee60 3fff6dcc 4024105e  
3fffee30:  3fff6c9c 3fff6ca0 00000006 00000001  
3fffee40:  4010383c 3fffc200 00000022 3fff6c9c  
3fffee50:  3fff4e8c 00000000 3fff6dcc 40241316  
3fffee60:  3fff4d74 3fff61bc 00000006 00000001  
3fffee70:  00000000 00000000 00000000 3fff4718  
3fffee80:  3fff6c9c 3fff4720 3fff471c 40286af8  
3fffee90:  0000003a 3b32a8c0 00000019 00000039  
3fffeea0:  00000000 00000019 3ffee250 4010130c  
3fffeeb0:  00000000 00000000 3ffecacf 3fff48f8  
3fffeec0:  3ffec9d6 3fff48f4 3fff4e8c 40284f31  
3fffeed0:  3fff4624 3fff69ec 3fff69ec 40100d0c  
3fffeee0:  00000008 3fff6624 3ffee250 3fff69ec  
3fffeef0:  3ffec9c8 00000000 3fff4e8c 40284399  
3fffef00:  0632a8c0 00000001 40105ae1 3ffee988  
3fffef10:  3ffedee0 401024c2 00000000 00000005  
3fffef20:  00000002 40102258 00000002 00000000  
3fffef30:  4010228d 00000002 00000000 3fff6a74  
3fffef40:  3fffdc80 00000000 3fff4e8c 4027a087  
3fffef50:  40000f49 3fffdab0 3fffdab0 40000f49  
3fffef60:  40000e19 00000005 000b7e74 00000000  
3fffef70:  00000000 aa55aa55 000000ed 40105c99  
3fffef80:  40105c9f 000b7e74 00000000 78f202d9  
3fffef90:  4010000d 3f798e95 66e4c054 6e85b772  
3fffefa0:  40272ec0 3fffef3c 40272e71 3fffff48  
3fffefb0:  3fffffc0 00000000 00000000 feefeffe  
3fffefc0:  feefeffe feefeffe feefeffe feefeffe  
3fffefd0:  feefeffe feefeffe feefeffe feefeffe  
3fffefe0:  feefeffe feefeffe feefeffe feefeffe  
3fffeff0:  feefeffe feefeffe feefeffe feefeffe  
3ffff000:  feefeffe feefeffe feefeffe feefeffe  
3ffff010:  feefeffe feefeffe feefeffe feefeffe  
3ffff020:  feefeffe feefeffe feefeffe feefeffe  
3ffff030:  feefeffe feefeffe feefeffe feefeffe  
3ffff040:  feefeffe feefeffe feefeffe feefeffe  
3ffff050:  feefeffe feefeffe feefeffe feefeffe  
3ffff060:  feefeffe feefeffe feefeffe feefeffe  
3ffff070:  feefeffe feefeffe feefeffe feefeffe  
3ffff080:  feefeffe feefeffe feefeffe feefeffe  
3ffff090:  feefeffe feefeffe feefeffe feefeffe  
3ffff0a0:  feefeffe feefeffe feefeffe feefeffe  
3ffff0b0:  feefeffe feefeffe feefeffe feefeffe  
3ffff0c0:  feefeffe feefeffe feefeffe feefeffe  
3ffff0d0:  feefeffe feefeffe feefeffe feefeffe  
3ffff0e0:  feefeffe feefeffe feefeffe feefeffe  
3ffff0f0:  feefeffe feefeffe feefeffe feefeffe  
3ffff100:  feefeffe feefeffe feefeffe feefeffe  
3ffff110:  feefeffe feefeffe feefeffe feefeffe  
3ffff120:  feefeffe feefeffe feefeffe feefeffe  
3ffff130:  feefeffe feefeffe feefeffe feefeffe  
3ffff140:  feefeffe feefeffe feefeffe feefeffe  
3ffff150:  feefeffe feefeffe feefeffe feefeffe  
3ffff160:  feefeffe feefeffe feefeffe feefeffe  
3ffff170:  feefeffe feefeffe feefeffe feefeffe  
3ffff180:  feefeffe feefeffe feefeffe feefeffe  
3ffff190:  feefeffe feefeffe feefeffe feefeffe  
3ffff1a0:  feefeffe feefeffe feefeffe feefeffe  
3ffff1b0:  feefeffe feefeffe feefeffe feefeffe  
3ffff1c0:  feefeffe feefeffe feefeffe feefeffe  
3ffff1d0:  feefeffe feefeffe feefeffe feefeffe  
3ffff1e0:  feefeffe feefeffe feefeffe feefeffe  
3ffff1f0:  feefeffe feefeffe feefeffe feefeffe  
3ffff200:  feefeffe feefeffe feefeffe feefeffe  
3ffff210:  feefeffe feefeffe feefeffe feefeffe  
3ffff220:  feefeffe feefeffe feefeffe feefeffe  
3ffff230:  feefeffe feefeffe feefeffe feefeffe  
3ffff240:  feefeffe feefeffe feefeffe feefeffe  
3ffff250:  feefeffe feefeffe feefeffe feefeffe  
3ffff260:  feefeffe feefeffe feefeffe feefeffe  
3ffff270:  feefeffe feefeffe feefeffe feefeffe  
3ffff280:  feefeffe feefeffe feefeffe feefeffe  
3ffff290:  feefeffe feefeffe feefeffe feefeffe  
3ffff2a0:  feefeffe feefeffe feefeffe feefeffe  
3ffff2b0:  feefeffe feefeffe feefeffe feefeffe  
3ffff2c0:  feefeffe feefeffe feefeffe feefeffe  
3ffff2d0:  feefeffe feefeffe feefeffe feefeffe  
3ffff2e0:  feefeffe feefeffe feefeffe feefeffe  
3ffff2f0:  feefeffe feefeffe feefeffe feefeffe  
3ffff300:  feefeffe feefeffe feefeffe feefeffe  
3ffff310:  feefeffe feefeffe feefeffe feefeffe  
3ffff320:  feefeffe feefeffe feefeffe feefeffe  
3ffff330:  feefeffe feefeffe feefeffe feefeffe  
3ffff340:  feefeffe feefeffe feefeffe feefeffe  
3ffff350:  feefeffe feefeffe feefeffe feefeffe  
3ffff360:  feefeffe feefeffe feefeffe feefeffe  
3ffff370:  feefeffe feefeffe feefeffe feefeffe  
3ffff380:  feefeffe feefeffe feefeffe feefeffe  
3ffff390:  feefeffe feefeffe feefeffe feefeffe  
3ffff3a0:  feefeffe feefeffe feefeffe feefeffe  
3ffff3b0:  feefeffe feefeffe feefeffe feefeffe  
3ffff3c0:  feefeffe feefeffe feefeffe feefeffe  
3ffff3d0:  feefeffe feefeffe feefeffe feefeffe  
3ffff3e0:  feefeffe feefeffe feefeffe feefeffe  
3ffff3f0:  feefeffe feefeffe feefeffe feefeffe  
3ffff400:  feefeffe feefeffe feefeffe feefeffe  
3ffff410:  feefeffe feefeffe feefeffe feefeffe  
3ffff420:  feefeffe feefeffe feefeffe feefeffe  
3ffff430:  feefeffe feefeffe feefeffe feefeffe  
3ffff440:  feefeffe feefeffe feefeffe feefeffe  
3ffff450:  feefeffe feefeffe feefeffe feefeffe  
3ffff460:  feefeffe feefeffe feefeffe feefeffe  
3ffff470:  feefeffe feefeffe feefeffe feefeffe  
3ffff480:  feefeffe feefeffe feefeffe feefeffe  
3ffff490:  feefeffe feefeffe feefeffe feefeffe  
3ffff4a0:  feefeffe feefeffe feefeffe feefeffe  
3ffff4b0:  feefeffe feefeffe feefeffe feefeffe  
3ffff4c0:  feefeffe feefeffe feefeffe feefeffe  
3ffff4d0:  feefeffe feefeffe feefeffe feefeffe  
3ffff4e0:  feefeffe feefeffe feefeffe feefeffe  
3ffff4f0:  feefeffe feefeffe feefeffe feefeffe  
3ffff500:  feefeffe feefeffe feefeffe feefeffe  
3ffff510:  feefeffe feefeffe feefeffe feefeffe  
3ffff520:  feefeffe feefeffe feefeffe feefeffe  
3ffff530:  feefeffe feefeffe feefeffe feefeffe  
3ffff540:  feefeffe feefeffe feefeffe feefeffe  
3ffff550:  feefeffe feefeffe feefeffe feefeffe  
3ffff560:  feefeffe feefeffe feefeffe feefeffe  
3ffff570:  feefeffe feefeffe feefeffe feefeffe  
3ffff580:  feefeffe feefeffe feefeffe feefeffe  
3ffff590:  feefeffe feefeffe feefeffe feefeffe  
3ffff5a0:  feefeffe feefeffe feefeffe feefeffe  
3ffff5b0:  feefeffe feefeffe feefeffe feefeffe  
3ffff5c0:  feefeffe feefeffe feefeffe feefeffe  
3ffff5d0:  feefeffe feefeffe feefeffe feefeffe  
3ffff5e0:  feefeffe feefeffe feefeffe feefeffe  
3ffff5f0:  feefeffe feefeffe feefeffe feefeffe  
3ffff600:  feefeffe feefeffe feefeffe feefeffe  
3ffff610:  feefeffe feefeffe feefeffe feefeffe  
3ffff620:  feefeffe feefeffe feefeffe feefeffe  
3ffff630:  feefeffe feefeffe feefeffe feefeffe  
3ffff640:  feefeffe feefeffe feefeffe feefeffe  
3ffff650:  feefeffe feefeffe feefeffe feefeffe  
3ffff660:  feefeffe feefeffe feefeffe feefeffe  
3ffff670:  feefeffe feefeffe feefeffe feefeffe  
3ffff680:  feefeffe feefeffe feefeffe feefeffe  
3ffff690:  feefeffe feefeffe feefeffe feefeffe  
3ffff6a0:  feefeffe feefeffe feefeffe feefeffe  
3ffff6b0:  feefeffe feefeffe feefeffe feefeffe  
3ffff6c0:  feefeffe feefeffe feefeffe feefeffe  
3ffff6d0:  feefeffe feefeffe feefeffe feefeffe  
3ffff6e0:  feefeffe feefeffe feefeffe feefeffe  
3ffff6f0:  feefeffe feefeffe feefeffe feefeffe  
3ffff700:  feefeffe feefeffe feefeffe feefeffe  
3ffff710:  feefeffe feefeffe feefeffe feefeffe  
3ffff720:  feefeffe feefeffe feefeffe feefeffe  
3ffff730:  feefeffe feefeffe feefeffe feefeffe  
3ffff740:  feefeffe feefeffe feefeffe feefeffe  
3ffff750:  feefeffe feefeffe feefeffe feefeffe  
3ffff760:  feefeffe feefeffe feefeffe feefeffe  
3ffff770:  feefeffe feefeffe feefeffe feefeffe  
3ffff780:  feefeffe feefeffe feefeffe feefeffe  
3ffff790:  feefeffe feefeffe feefeffe feefeffe  
3ffff7a0:  feefeffe feefeffe feefeffe feefeffe  
3ffff7b0:  feefeffe feefeffe feefeffe feefeffe  
3ffff7c0:  feefeffe feefeffe feefeffe feefeffe  
3ffff7d0:  feefeffe feefeffe feefeffe feefeffe  
3ffff7e0:  feefeffe feefeffe feefeffe feefeffe  
3ffff7f0:  feefeffe feefeffe feefeffe feefeffe  
3ffff800:  feefeffe feefeffe feefeffe feefeffe  
3ffff810:  feefeffe feefeffe feefeffe feefeffe  
3ffff820:  feefeffe feefeffe feefeffe feefeffe  
3ffff830:  feefeffe feefeffe feefeffe feefeffe  
3ffff840:  feefeffe feefeffe feefeffe feefeffe  
3ffff850:  feefeffe feefeffe feefeffe feefeffe  
3ffff860:  feefeffe feefeffe feefeffe feefeffe  
3ffff870:  feefeffe feefeffe feefeffe feefeffe  
3ffff880:  00000000 00000000 0000000a 00000000  
3ffff890:  00000000 00000000 0000000a 00000000  
3ffff8a0:  feefeffe feefeffe feefeffe feefeffe  
3ffff8b0:  00000000 a0000000 00000000 0000001c  
3ffff8c0:  00000000 feefeffe 00000000 00000000  
3ffff8d0:  3ffffa30 00000000 3ffffa30 4025e636  
3ffff8e0:  0000a000 3ffff9d3 feefeffe feefeffe  
3ffff8f0:  feefeffe feefeffe 4025ef6d 00000001  
3ffff900:  3ffffa30 00000001 3ffffa30 4025e636  
3ffff910:  3ffff990 3ffff9d2 3ffff940 00000000  
3ffff920:  feefeffe feefeffe feefeffe 4025e56c  
3ffff930:  3ffffa30 3ffff9d2 3ffff990 4025f098  
3ffff940:  00000001 feefeffe feefeffe feefeffe  
3ffff950:  3ffe8344 00000000 0000000a 402aba49  
3ffff960:  3ffff9d3 00000000 00000000 feefeffe  
3ffff970:  feefeffe feefeffe feefeffe 3ffe904a  
3ffff980:  00000000 3ffe9049 3ffffa30 4025ea6b  
3ffff990:  00000000 ffffffff ffffffff 00000000  
3ffff9a0:  00000001 00000001 fe302069 feefeffe  
3ffff9b0:  feefeffe feefeffe feefeffe feefeffe  
3ffff9c0:  feefeffe feefeffe feefeffe feefeffe  
3ffff9d0:  00303231 feefeffe feefeffe feefeffe  
3ffff9e0:  feefeffe feefeffe 00000000 feefeffe  
3ffff9f0:  3ffffac0 3ffffab0 00000010 3ffe8344  
3ffffa00:  00000001 00000009 00000001 feefeffe  
3ffffa10:  00000468 feefeffe feefeffe 00000008  
3ffffa20:  007a1200 31fda03a 00000300 4025b6c1  
3ffffa30:  3ffffb06 4024e43c 000003f9 ffff0208  
3ffffa40:  00000000 00000000 00000000 3ffffaa0  
3ffffa50:  3fff01a8 3fff01cc 00000000 40253aa8  
3ffffa60:  00000062 026ca891 3ffeec20 40254502  
3ffffa70:  3ffeb040 3fff01cc 00000000 40230ed8  
3ffffa80:  00000062 026ca891 40103b1a 40218b08  
3ffffa90:  3ffeb040 00000006 00000000 402193bd  
3ffffaa0:  0000000c 00000014 3ffffb00 00000000  
3ffffab0:  3ffffaa0 00000000 0000000a 00000000  
3ffffac0:  00000000 00000000 0000000a 00000000  
3ffffad0:  3ffe9058 00000000 3fffff00 00000008  
3ffffae0:  3fff01a8 00000000 3fffff00 402196d0  
3ffffaf0:  00000000 00000008 00002000 00000000  
3ffffb00:  38007776 38003000 3ffffc00 4025e636  
3ffffb10:  0000a000 3ffffc03 3ffffc73 3ffffc30  
3ffffb20:  00000005 00000000 4025ef6d 00000007  
3ffffb30:  3ffffc60 00000007 3ffffc60 4025e636  
3ffffb40:  3ffffbc0 3ffffbfc 3ffffb70 00000000  
3ffffb50:  00000000 a0000000 00000000 4025e56c  
3ffffb60:  3ffffc60 3ffffbfc 3ffffbc0 4025f098  
3ffffb70:  00000007 00000000 3ffffcd0 00000007  
3ffffb80:  3ffe8344 00000000 0000000a 402aba49  
3ffffb90:  3ffffc03 00000002 00000000 00000002  
3ffffba0:  3ffffcd0 00000002 3ffffcd0 402acb99  
3ffffbb0:  00000000 402acb98 3ffffc60 4025ea6b  
3ffffbc0:  00000005 00000000 00000020 40100d0c  
3ffffbd0:  00000000 00000000 0000001f 40100d0c  
3ffffbe0:  3ffea7c5 40105b93 3fffc228 40106439  
3ffffbf0:  4000050c 3ffee1d8 00000020 40100d0c  
3ffffc00:  00000000 00000000 0000001f 40100d0c  
3ffffc10:  00000005 00000000 00000020 40100d0c  
3ffffc20:  4000050c 00000001 00000005 401028e0  
3ffffc30:  3ffea7c5 40105b93 3ffee188 fffffffe  
3ffffc40:  40103477 3ffee188 3ff1fe00 00000000  
3ffffc50:  00000005 00000000 00000020 40100d0c  
3ffffc60:  00000005 00000000 00000020 40100d0c  
3ffffc70:  3ffea7c5 40105b93 00000005 401028e0  
3ffffc80:  3ffea7c5 40105b93 3ffee200 40100d0c  
3ffffc90:  40103477 3ffee200 3ffeec20 40103658  
3ffffca0:  00000005 00000000 00000020 40100d0c  
3ffffcb0:  3ffeb040 00000000 00000005 401028e0  
3ffffcc0:  3ffea7c5 40105b93 3ffee188 00000100  
3ffffcd0:  40103477 3ffee188 00002200 00000001  
3ffffce0:  00000060 05045742 3ffeec20 40103658  
3ffffcf0:  3ffeb07c 00000000 00000000 401028e0  
3ffffd00:  00000060 05045742 40103b1a 00000100  
3ffffd10:  3ffeb07c 7fffffff 00002200 00000001  
3ffffd20:  00000001 00006208 00000005 401028e0  
3ffffd30:  3ffeb07c 40105b93 3ffee188 05045742  
3ffffd40:  00000005 00000000 00000020 40100d0c  
3ffffd50:  00000005 00000000 00000005 401028e0  
3ffffd60:  3ffea7c5 40105b93 3ffee188 401028e0  
3ffffd70:  40103477 3ffee188 00000020 40100d0c  
3ffffd80:  00000005 00000000 00000020 40100d0c  
3ffffd90:  00000005 00000000 00000005 401028e0  
3ffffda0:  3ffea7c5 40105b93 3ffee250 401028e0  
3ffffdb0:  40103477 3ffee250 00000020 40100d0c  
3ffffdc0:  00007fff 0504c038 3ffeec20 40103658  
3ffffdd0:  3ffeb07c 00000000 00000000 00000001  
3ffffde0:  00007fff 0504c038 40103b1a 00000100  
3ffffdf0:  3ffeb07c 7fffffff 00002200 00000001  
3ffffe00:  00000001 00004288 3fff6198 40252b39  
3ffffe10:  3ffeb07c 3ffffe40 0000002c 0504c038  
3ffffe20:  3ffeb070 2c9f0300 4000050c 3fffc278  
3ffffe30:  4010383c 3fffc200 00000022 40231ce8  
3ffffe40:  40228f5e 00000030 00000010 ffffffff  
3ffffe50:  40235588 00000000 00000000 00000001  
3ffffe60:  00004bc6 00000001 00000000 fffffffe  
3ffffe70:  00000000 3fffc6fc 00000000 3fff3651  
3ffffe80:  3fff00f2 3fffdad0 3fff3c9c 00000030  
3ffffe90:  00000000 00000000 3fff3c5c 402380eb  
3ffffea0:  ffffffff 3fffc6fc 00000001 3fff2a48  
3ffffeb0:  00000000 4bc6a7f0 000148e5 00000003  
3ffffec0:  00000000 00000000 4bc6a7f0 00000000  
3ffffed0:  00000000 00000000 40101086 845a1cac  
3ffffee0:  00000000 4bc6a7f0 40101086 40213d0c  
3ffffef0:  00000001 3fff3a1c 3fff6288 40100b13  
3fffff00:  00000001 4bc6a7f0 3fff6254 4024c034  
3fffff10:  00000000 4bc6a7f0 000148e5 3fff3c9c  
3fffff20:  007a1200 95b0eb8d 4bc6a700 3fff3c9c  
3fffff30:  00000000 00000000 00000001 40100d0c  
3fffff40:  3fffdad0 3fff00f2 3fff2a48 3fff00f2  
3fffff50:  3fffdad0 3fff3c9c 40253ad3 3fffefa0  
3fffff60:  3fffdad0 3fff00f2 3fff2a48 4023558e  
3fffff70:  00000000 feefeffe 00efeffe feefeffe  
3fffff80:  00000000 00000000 00000001 3fff3c9c  
3fffff90:  3fffdad0 00000000 3fff3c5c 402360cc  
3fffffa0:  feefeffe feefeffe feefeffe 40253bc0  
<<<stack<<<
 
--------------- CUT HERE FOR EXCEPTION DECODER ---------------
 
 ets Jan  8 2013,rst cause:2, boot mode:(3,6)
 
load 0x4010f000, len 3584, room 16 
tail 0
chksum 0xb0
csum 0xb0
v3969889e
~ld
Ada
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ewoudwijma** [comment]
What environment settings are you using?

Not thoroughly tested but with these settings I am able to run it:

[env:esp8266_4MB_min]
extends = env:d1_mini
upload_speed = 460800 ;115200
build_flags = ${common.build_flags_esp8266}
  -D WLED_RELEASE_NAME=esp8266_4MB_min
  -D WLED_DISABLE_ALEXA
  -D WLED_DISABLE_BLYNK
  -D WLED_DISABLE_HUESYNC
  ; -D WLED_DISABLE_2D
  -UWLED_USE_MY_CONFIG

**Comment by blazoncek** [comment]
IMO faulty flash or incorrect flash mode. Use DOUT flash mode.

**Comment by The-Techy** [comment]
> What environment settings are you using?
> 
> Not thoroughly tested but with these settings I am able to run latest WLED:
> 
> [env:esp8266_4MB_min] extends = env:d1_mini upload_speed = 460800 ;115200 build_flags = ${common.build_flags_esp8266} -D WLED_RELEASE_NAME=esp8266_4MB_min -D WLED_DISABLE_ALEXA -D WLED_DISABLE_BLYNK -D WLED_DISABLE_HUESYNC ; -D WLED_DISABLE_2D -UWLED_USE_MY_CONFIG
> 
> Or try the compiled bin for this. See [here](https://github.com/srg74/WLED-wemos-shield/blob/master/resources/experimental/Firmware/AudioReactive/WLEDMM_0.14.0.8_esp8266_4MB_min.bin) (MM version)

I've tried using both the precompiled versions, and compiling my own (pretty much just select the enviroment as d1_mini and don't change anything else)

**Comment by The-Techy** [comment]
> IMO faulty flash or incorrect flash mode. Use DOUT flash mode.

How do you change the flash mode? I didn't realise there were different ones

**Comment by blazoncek** [comment]
using esptool (`-fm dout`).

**Comment by The-Techy** [comment]
> using esptool (`-fm dout`).

Sorry for the late response, I've been a bit busy. I don't use esptool, but in vscode I added `board_build.flash_mode = dout` to the d1_mini env, which as I understand, should force it to use dout. I still get the same issue, it crashes a little slower now, but still crashes. Even tried updating it OTA with one of the precompiled bins, and still same issue. I've tried it on two boards now, both with the same issue. Could it be a faulty batch of boards?

**Comment by The-Techy** [comment]
Just tried uploading it onto a board I got a couple months before the others, same seller, and it has the exact same issue. I therefore don't think it's an issue with the boards, since they all run WLED 0.12.0 perfectly. Also tried an ESP32 (again, same seller), which also ran 0.12.0 absolutely fine, and the identical issue happened.

TLDR: Tried almost every board I have (all from same seller, different purchase times), all with same crashing issue. They all worked fine on 0.12.0.

**Comment by The-Techy** [comment]
Okay, just tried esptool with the exact same results. Even erased all the flash, didn't work. I can't even use AP mode, since it crashes there too. Really annoying.

**Comment by blazoncek** [comment]
Perhaps consider getting a new ESP from another seller, diffrent brand, different model, etc.
8266 costs $2 or thereabouts.

**Comment by The-Techy** [comment]
So, it appears as though I've been very stupid. I tried changing everything, cable, board, upload method, etc. EXCEPT, the device I was accessing the UI through! All this time I've been using my laptop to connect, but I tried my phone, and I can change the lights, no issue, without it crashing!

Now to figure out why my computer would disconnect every time. Any ideas?

**Comment by The-Techy** [comment]
After some more testing, it's definitely just macOS that's the problem. Through bootcamp, on the same machine, wled works fine, it's only in macOS. Also, I've tried multiple browsers, and a wled desktop app, all with the same issue. What could be causing macOS to do this?

**Comment by blazoncek** [comment]
Please close as this is apparently not a WLED issue after all.
